units/host_status.go

```diff
@@ -312,3 +300,21 @@ func (j *cloudHostReadyJob) logHostStatusMessage(h *host.Host, cloudStatus cloud
 		})
 	}
 }
+
+// prepareToTerminateHost handles a host that is an unrecoverable state by
+// marking it for termination and handling any other.
```

Comment:
did you mean to add to the end of this comment?

Reply:
Yeah it should have more words at the end, I fixed it.